angular/projects/my-lib/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")
+load("@rules_angular//src/architect:ng_library.bzl", "ng_library")
+load("@rules_angular//src/architect:ng_test.bzl", "ng_test")
+
+package(default_visibility = ["//visibility:public"])
+
+copy_to_bin(
+    name = "srcs",
+    srcs = glob(
+        ["src/**/*"],
+        exclude = ["dist/"],
+    ),
+)
+
+ng_library(
+    name = "my-lib",
+    srcs = [":srcs"],
+    ng_config = "//angular:ng-config",
+    node_modules = "//angular:node_modules",
+)
+
+ng_test(
+    name = "test",
+    size = "small",
+    srcs = [":srcs"],
```

Comment:
We need to pass the pure .ts and can't pass `:my-lib`?

Reply:
correct, my-lib produces an ng-packagr output format intended to publish to npm.

Again, our goal here is just to run Angular CLI the same way you would outside Bazel. Should be functionally equivalent to how Nx does it. I'd expect karma to read the application sources.